### ISSUE TITLE
Missing hvac python module for hashi_vault lookup filter

### ISSUE THREAD

**Issue by penoux** [ansible/awx#2827]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - Ansible

##### SUMMARY

The HashiCorp Vault external credential type is not working, despite a correct configuration, complaining for a missing python dependency : `hvac`.

##### ENVIRONMENT
* AWX version: 2.1.0
* AWX install method: docker on linux
* Ansible version:  2.7.2
* Operating System: CentOS 7
* Web Browser: Firefox

##### STEPS TO REPRODUCE

Follow the example setup for HashiCorp Vault external credentials: https://www.ansible.com/blog/ansible-tower-feature-spotlight-custom-credentials. 

##### EXPECTED RESULTS

Lookup succeeds.

##### ACTUAL RESULTS

The `hashi_vault` lookup filter fails with the following error.

```
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'hashi_vault'. Error was a <class 'ansible.errors.AnsibleError'>, original message: Please pip install hvac to use the hashi_vault lookup module."}
```

##### ADDITIONAL INFORMATION

When I manually install `hvac` in the awx_task container (in ansible venv), all is OK and the lookup succeeds. The module `hvac` should probably be added in ansible requirements for the image build.

**Comment by jakemcdermott** [comment]
Hey penoux, thanks for creating this issue.

At this time, we do not provide dependencies for the entire set of shipped-with-Ansible modules in the environment.

The normal way to do this is to create a custom image that contains the updated libraries, system dependencies, etc.

You can also try mounting a [custom virtualenv](https://github.com/ansible/awx/blob/devel/docs/custom_virtualenvs.md) into your container at install time, and configuring it for use with specific job templates.

If you have any questions, please reach out to our community mailing list or stop by our irc channel on freenode #ansible-awx.

**Comment by penoux** [comment]
Hello Jake,

I perfectly understand that the aim of AWX is not to provide all set of Ansible modules dependencies.

In this case though, I raised the issue because the feature and its setup (with example) are fully documented in Tower official doc: https://www.ansible.com/blog/ansible-tower-feature-spotlight-custom-credentials.

Our AWX instance is a mutualized one (over dozens of teams), the team in charge uses standard image, that's why I suggested an inclusion on standard `requirements.in`.

Regards
